### ISSUE TITLE
2025-03-13, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#109]
# Today's Arxiv Papers

(1) [MoC: Mixtures of Text Chunking Learners for Retrieval-Augmented Generation System](http://arxiv.org/abs/2503.09600v1), 2025-03-12

(2) [How to Protect Yourself from 5G Radiation? Investigating LLM Responses to Implicit Misinformation](http://arxiv.org/abs/2503.09598v1), 2025-03-12

(3) [SimLingo: Vision-Only Closed-Loop Autonomous Driving with Language-Action Alignment](http://arxiv.org/abs/2503.09594v1), 2025-03-12

(4) [BIMBA: Selective-Scan Compression for Long-Range Video Question Answering](http://arxiv.org/abs/2503.09590v1), 2025-03-12

(5) [Cost-Optimal Grouped-Query Attention for Long-Context LLMs](http://arxiv.org/abs/2503.09579v1), 2025-03-12

(6) [Plan-and-Act: Improving Planning of Agents for Long-Horizon Tasks](http://arxiv.org/abs/2503.09572v1), 2025-03-12

(7) [Towards Reasoning Era: A Survey of Long Chain-of-Thought for Reasoning Large Language Models](http://arxiv.org/abs/2503.09567v1), 2025-03-12

(8) [Large Language Models for Multi-Facility Location Mechanism Design](http://arxiv.org/abs/2503.09533v1), 2025-03-12

(9) [Search-R1: Training LLMs to Reason and Leverage Search Engines with Reinforcement Learning](http://arxiv.org/abs/2503.09516v1), 2025-03-12

(10) [ReMA: Learning to Meta-think for LLMs with Multi-Agent Reinforcement Learning](http://arxiv.org/abs/2503.09501v1), 2025-03-12

(11) [BAMBI: Developing Baby Language Models for Italian](http://arxiv.org/abs/2503.09481v1), 2025-03-12

(12) [Explicit Learning and the LLM in Machine Translation](http://arxiv.org/abs/2503.09454v1), 2025-03-12

(13) [Florenz: Scaling Laws for Systematic Generalization in Vision-Language Models](http://arxiv.org/abs/2503.09443v1), 2025-03-12

(14) [PromptMap: An Alternative Interaction Style for AI-Based Image Generation](http://dx.doi.org/10.1145/3708359.3712150), 2025-03-12

(15) [CASTLE: Benchmarking Dataset for Static Code Analyzers and LLMs towards CWE Detection](http://arxiv.org/abs/2503.09433v1), 2025-03-12

(16) [Got Compute, but No Data: Lessons From Post-training a Finnish LLM](http://arxiv.org/abs/2503.09407v1), 2025-03-12

(17) [Towards Next-Generation Recommender Systems: A Benchmark for Personalized Recommendation Assistant with LLMs](http://arxiv.org/abs/2503.09382v1), 2025-03-12

(18) [RetSTA: An LLM-Based Approach for Standardizing Clinical Fundus Image Reports](http://arxiv.org/abs/2503.09358v1), 2025-03-12

(19) [Safer or Luckier? LLMs as Safety Evaluators Are Not Robust to Artifacts](http://arxiv.org/abs/2503.09347v1), 2025-03-12

(20) [An Evaluation of LLMs for Detecting Harmful Computing Terms](http://arxiv.org/abs/2503.09341v1), 2025-03-12

(21) [NVP-HRI: Zero Shot Natural Voice and Posture-based Human-Robot Interaction via Large Language Model](http://dx.doi.org/10.1016/j.eswa.2024.126360), 2025-03-12

(22) [CyberLLMInstruct: A New Dataset for Analysing Safety of Fine-Tuned LLMs Using Cyber Security Data](http://arxiv.org/abs/2503.09334v1), 2025-03-12

(23) [A Survey on Enhancing Causal Reasoning Ability of Large Language Models](http://arxiv.org/abs/2503.09326v1), 2025-03-12

(24) [xVLM2Vec: Adapting LVLM-based embedding models to multilinguality using Self-Knowledge Distillation](http://arxiv.org/abs/2503.09313v1), 2025-03-12

(25) [Adaptive political surveys and GPT-4: Tackling the cold start problem with simulated user interactions](http://arxiv.org/abs/2503.09311v1), 2025-03-12

(26) [Priority-Aware Preemptive Scheduling for Mixed-Priority Workloads in MoE Inference](http://dx.doi.org/10.1145/3721146.3721956), 2025-03-12

(27) [Prompt Inference Attack on Distributed Large Language Model Inference Frameworks](http://arxiv.org/abs/2503.09291v1), 2025-03-12

(28) [Crowdsourced Homophily Ties Based Graph Annotation Via Large Language Model](http://arxiv.org/abs/2503.09281v1), 2025-03-12

(29) [Fine-Tuning Large Language Models for Educational Support: Leveraging Gagne's Nine Events of Instruction for Lesson Planning](http://arxiv.org/abs/2503.09276v1), 2025-03-12

(30) [COLA: A Scalable Multi-Agent Framework For Windows UI Task Automation](http://arxiv.org/abs/2503.09263v1), 2025-03-12

(31) [DeepInnovation AI: A Global Dataset Mapping the AI innovation and technology Transfer from Academic Research to Industrial Patents](http://arxiv.org/abs/2503.09257v1), 2025-03-12

(32) [LREF: A Novel LLM-based Relevance Framework for E-commerce](http://dx.doi.org/10.1145/3701716.3715246), 2025-03-12

(33) [Rethinking Prompt-based Debiasing in Large Language Models](http://arxiv.org/abs/2503.09219v1), 2025-03-12

(34) [Evaluating the Generalizability of LLMs in Automated Program Repair](http://arxiv.org/abs/2503.09217v1), 2025-03-12

(35) [Why LLMs Cannot Think and How to Fix It](http://arxiv.org/abs/2503.09211v1), 2025-03-12

(36) [Quality Over Quantity? LLM-Based Curation for a Data-Efficient Audio-Video Foundation Model](http://arxiv.org/abs/2503.09205v1), 2025-03-12

(37) [Token Weighting for Long-Range Language Modeling](http://arxiv.org/abs/2503.09202v1), 2025-03-12

(38) [FaVChat: Unlocking Fine-Grained Facail Video Understanding with Multimodal Large Language Models](http://arxiv.org/abs/2503.09158v1), 2025-03-12

(39) [Is LLMs Hallucination Usable? LLM-based Negative Reasoning for Fake News Detection](http://arxiv.org/abs/2503.09153v1), 2025-03-12

(40) [AdaptAI: A Personalized Solution to Sense Your Stress, Fix Your Mess, and Boost Productivity](http://arxiv.org/abs/2503.09150v1), 2025-03-12

(41) [Generative Frame Sampler for Long Video Understanding](http://arxiv.org/abs/2503.09146v1), 2025-03-12

(42) [Exo2Ego: Exocentric Knowledge Guided MLLM for Egocentric Video Understanding](http://arxiv.org/abs/2503.09143v1), 2025-03-12

(43) [GRU: Mitigating the Trade-off between Unlearning and Retention for Large Language Models](http://arxiv.org/abs/2503.09117v1), 2025-03-12

(44) [VaxGuard: A Multi-Generator, Multi-Type, and Multi-Role Dataset for Detecting LLM-Generated Vaccine Misinformation](http://arxiv.org/abs/2503.09103v1), 2025-03-12

(45) [Multi-Modal Foundation Models for Computational Pathology: A Survey](http://arxiv.org/abs/2503.09091v1), 2025-03-12

(46) [LocAgent: Graph-Guided LLM Agents for Code Localization](http://arxiv.org/abs/2503.09089v1), 2025-03-12

(47) [Probing Latent Subspaces in LLM for AI Security: Identifying and Manipulating Adversarial States](http://arxiv.org/abs/2503.09066v1), 2025-03-12

(48) [ManeuverGPT Agentic Control for Safe Autonomous Stunt Maneuvers](http://arxiv.org/abs/2503.09035v1), 2025-03-12

(49) [Teaching LLMs How to Learn with Contextual Fine-Tuning](http://arxiv.org/abs/2503.09032v1), 2025-03-12

(50) [DAST: Difficulty-Aware Self-Training on Large Language Models](http://arxiv.org/abs/2503.09029v1), 2025-03-12

(51) [Measure Twice, Cut Once: Grasping Video Structures and Event Semantics with LLMs for Video Temporal Localization](http://arxiv.org/abs/2503.09027v1), 2025-03-12

(52) [Aligning to What? Limits to RLHF Based Alignment](http://arxiv.org/abs/2503.09025v1), 2025-03-12

(53) [Prompt Inversion Attack against Collaborative Inference of Large Language Models](http://arxiv.org/abs/2503.09022v1), 2025-03-12

(54) [Enhancing High-Quality Code Generation in Large Language Models with Comparative Prefix-Tuning](http://arxiv.org/abs/2503.09020v1), 2025-03-12

(55) [Leveraging Retrieval Augmented Generative LLMs For Automated Metadata Description Generation to Enhance Data Catalogs](http://arxiv.org/abs/2503.09003v1), 2025-03-12

(56) [KNighter: Transforming Static Analysis with LLM-Synthesized Checkers](http://arxiv.org/abs/2503.09002v1), 2025-03-12

(57) [JBFuzz: Jailbreaking LLMs Efficiently and Effectively Using Fuzzing](http://arxiv.org/abs/2503.08990v1), 2025-03-12

(58) [I Predict Therefore I Am: Is Next Token Prediction Enough to Learn Human-Interpretable Concepts from Data?](http://arxiv.org/abs/2503.08980v1), 2025-03-12

(59) [Large Language Models-Aided Program Debloating](http://arxiv.org/abs/2503.08969v1), 2025-03-12

(60) [LLM-Driven Usefulness Labeling for IR Evaluation](http://arxiv.org/abs/2503.08965v1), 2025-03-12

(61) [Gradient-guided Attention Map Editing: Towards Efficient Contextual Hallucination Mitigation](http://arxiv.org/abs/2503.08963v1), 2025-03-11

(62) [ARCHED: A Human-Centered Framework for Transparent, Responsible, and Collaborative AI-Assisted Instructional Design](http://arxiv.org/abs/2503.08931v1), 2025-03-11

(63) [Enhancing Large Language Models for Hardware Verification: A Novel SystemVerilog Assertion Dataset](http://arxiv.org/abs/2503.08923v1), 2025-03-11

(64) [Backtracking for Safety](http://arxiv.org/abs/2503.08919v1), 2025-03-11

(65) [Interpreting the Repeated Token Phenomenon in Large Language Models](http://arxiv.org/abs/2503.08908v1), 2025-03-11

(66) [Seeing What's Not There: Spurious Correlation in Multimodal LLMs](http://arxiv.org/abs/2503.08884v1), 2025-03-11

(67) [LLMs Know What to Drop: Self-Attention Guided KV Cache Eviction for Efficient Long-Context Inference](http://arxiv.org/abs/2503.08879v1), 2025-03-11

(68) [Interpretable and Robust Dialogue State Tracking via Natural Language Summarization with LLMs](http://arxiv.org/abs/2503.08857v1), 2025-03-11

(69) [Contrastive Speaker-Aware Learning for Multi-party Dialogue Generation with LLMs](http://arxiv.org/abs/2503.08842v1), 2025-03-11

(70) [ResBench: Benchmarking LLM-Generated FPGA Designs with Resource Awareness](http://arxiv.org/abs/2503.08823v1), 2025-03-11

(71) [Cross-Examiner: Evaluating Consistency of Large Language Model-Generated Explanations](http://arxiv.org/abs/2503.08815v1), 2025-03-11

(72) [Robust Multi-Objective Controlled Decoding of Large Language Models](http://arxiv.org/abs/2503.08796v1), 2025-03-11

(73) [Randomness, Not Representation: The Unreliability of Evaluating Cultural Alignment in LLMs](http://arxiv.org/abs/2503.08688v1), 2025-03-11

(74) [Perplexity Trap: PLM-Based Retrievers Overrate Low Perplexity Documents](http://arxiv.org/abs/2503.08684v1), 2025-03-11

(75) [CoLMDriver: LLM-based Negotiation Benefits Cooperative Autonomous Driving](http://arxiv.org/abs/2503.08683v1), 2025-03-11

(76) [Self-Taught Self-Correction for Small Language Models](http://arxiv.org/abs/2503.08681v1), 2025-03-11

(77) [Exploring the Word Sense Disambiguation Capabilities of Large Language Models](http://arxiv.org/abs/2503.08662v1), 2025-03-11

(78) [Exploiting Instruction-Following Retrievers for Malicious Information Retrieval](http://arxiv.org/abs/2503.08644v1), 2025-03-11

(79) [LightGen: Efficient Image Generation through Knowledge Distillation and Direct Preference Optimization](http://arxiv.org/abs/2503.08619v1), 2025-03-11

(80) [EMMOE: A Comprehensive Benchmark for Embodied Mobile Manipulation in Open Environments](http://arxiv.org/abs/2503.08604v1), 2025-03-11

(81) [NSF-SciFy: Mining the NSF Awards Database for Scientific Claims](http://arxiv.org/abs/2503.08600v1), 2025-03-11

(82) [HierarQ: Task-Aware Hierarchical Q-Former for Enhanced Video Understanding](http://arxiv.org/abs/2503.08585v1), 2025-03-11

(83) [Chatbots for Data Collection in Surveys: A Comparison of Four Theory-Based Interview Probes](http://dx.doi.org/10.1145/3706598.3714128), 2025-03-11

(84) [RAG-Adapter: A Plug-and-Play RAG-enhanced Framework for Long Video Understanding](http://arxiv.org/abs/2503.08576v1), 2025-03-11

(85) [DeepReview: Improving LLM-based Paper Review with Human-like Deep Thinking Process](http://arxiv.org/abs/2503.08569v1), 2025-03-11

(86) [Reasoning and Sampling-Augmented MCQ Difficulty Prediction via LLMs](http://arxiv.org/abs/2503.08551v1), 2025-03-11

(87) [Transferring Extreme Subword Style Using Ngram Model-Based Logit Scaling](http://arxiv.org/abs/2503.08550v1), 2025-03-11

(88) [Graph of AI Ideas: Leveraging Knowledge Graphs and LLMs for AI Research Idea Generation](http://arxiv.org/abs/2503.08549v1), 2025-03-11

(89) [DAFE: LLM-Based Evaluation Through Dynamic Arbitration for Free-Form Question-Answering](http://arxiv.org/abs/2503.08542v1), 2025-03-11

(90) [Mellow: a small audio language model for reasoning](http://arxiv.org/abs/2503.08540v1), 2025-03-11

(91) [Desirable Unfamiliarity: Insights from Eye Movements on Engagement and Readability of Dictation Interfaces](http://arxiv.org/abs/2503.08539v1), 2025-03-11

(92) [Chemical reasoning in LLMs unlocks steerable synthesis planning and reaction mechanism elucidation](http://arxiv.org/abs/2503.08537v1), 2025-03-11

(93) [GTR: Guided Thought Reinforcement Prevents Thought Collapse in RL-based VLM Agent Training](http://arxiv.org/abs/2503.08525v1), 2025-03-11

(94) [Position-Aware Depth Decay Decoding ($D^3$): Boosting Large Language Model Inference Efficiency](http://arxiv.org/abs/2503.08524v1), 2025-03-11

(95) [LightPlanner: Unleashing the Reasoning Capabilities of Lightweight Large Language Models in Task Planning](http://arxiv.org/abs/2503.08508v1), 2025-03-11

(96) [Referring to Any Person](http://arxiv.org/abs/2503.08507v1), 2025-03-11

(97) [ReviewAgents: Bridging the Gap Between Human and AI-Generated Paper Reviews](http://arxiv.org/abs/2503.08506v1), 2025-03-11

(98) [Enhancing Multi-Hop Fact Verification with Structured Knowledge-Augmented Large Language Models](http://arxiv.org/abs/2503.08495v1), 2025-03-11

(99) [FastCache: Optimizing Multimodal LLM Serving through Lightweight KV-Cache Compression Framework](http://arxiv.org/abs/2503.08461v1), 2025-03-11

(100) [KAP: MLLM-assisted OCR Text Enhancement for Hybrid Retrieval in Chinese Non-Narrative Documents](http://arxiv.org/abs/2503.08452v1), 2025-03-11
